src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/CatalogPromotion.xml

```diff
@@ -80,6 +89,10 @@ Example configuration for `percentage_discount` action type:
                 <attribute name="method">GET</attribute>
                 <attribute name="normalization_context">
                     <attribute name="groups">admin:catalog_promotion:read</attribute>
+
+                    <!-- We've got to turn off this option here, as it auto-turned on when adding PATCH itemOperations, which would do BC.-->
```

Comment:
Why is it necessary in the current PR? Was it broken somehow? Do tests need to be updated for this change?

Reply:
API Platform automatically hides all null-valued fields in GET endpoints, when any PATCH is added to the resource. This makes a BC, because in Sylius, especially in this resource we allow to display nullable values. But when I added the PATCH, the PHPUnit tests were red becouse of missing fields in responses.